test/test-BlueprintParser.cc

```diff
@@ -712,7 +727,146 @@ TEST_CASE("Report error when data Structure inheritance graph contains a cycle",
     SectionParserHelper<Blueprint, BlueprintParser>::parse(source, BlueprintSectionType, blueprint, ExportSourcemapOption, Models(), &blueprint);
 
     REQUIRE(blueprint.report.error.code == MSONError);
-    SourceMapHelper::check(blueprint.report.error.location, 28, 9);
+    SourceMapHelper::check(blueprint.report.error.location, 19, 9);
```

Comment:
Previously, parser used to resolve all the inheritance tables before returning error which is basically the last error occurring. Now, the parser returns error as soon as it encounters when resolving the inheritance tables. https://github.com/apiaryio/snowcrash/pull/317/files#diff-da3aa8ccabc921ae656c97332b64e901R425